tests/fixtures/tests.py

```diff
@@ -224,6 +224,11 @@ def test_dumpdata_with_excludes(self):
                 "Unknown model in excludes: fixtures.FooModel"):
             self._dumpdata_assert(['fixtures', 'sites'], '', exclude_list=['fixtures.FooModel'])
 
+    def test_load_fixture_with_special_characters(self):
+        Site.objects.all().delete()
```

Comment:
Would be slightly better to use non-contrib models in case sites is someday removed.